recipes-kernel/linux/4.14/linux-openxt_4.14.22.bb

```diff
@@ -3,6 +3,7 @@ DESCRIPTION = "Linux kernel for OpenXT service VMs."
 # Use the one from meta-openembedded/meta-oe
 require recipes-kernel/linux/linux.inc
 require recipes-kernel/linux/linux-openxt.inc
+require recipes-kernel/linux/linux-openxt-4.14.inc
```

Comment:
Do you want to just put add linux-openxt.inc?  It'll be required for all version going forward, and the code is conditioned on the existence of the objtool in the first place.

Also, objtool depends on the elfutils to build objtool.
"Cannot use CONFIG_STACK_VALIDATION, please install libelf-dev, libelf-devel or elfutils-libelf-devel"
DEPENDS += "elfutils-native" didn't work, and I ended up installing libelf-dev on the build host.

Reply:
> Do you want to just put add linux-openxt.inc

As I took the patch from patchwork directly, I expect this will make its way upstream sooner rather than later. Splitting it in `linux-openxt-4.14.inc` was to make the revert easier.

`libelf-dev` is already in the [dependency requirement list](https://github.com/OpenXT/openxt/blob/master/build-scripts/oe/setup.sh#L35) for the environment to build OpenXT. I believe it was added for that reason when the changes for UEFI support introduced the 64bit kernel. I do not remember why the native recipe does not cut it either.

Reply:
I was surprised pyro didn't have the patch already, but pyro doesn't support Linux 4.14.

I think I searched for the wrong thing, elfutils, in the OpenXT PKGS var, which naturally failed. Good to see it already listed.

Reply:
> I was surprised pyro didn't have the patch already

Me too! Even `sumo` does not have it yet :).